cde/sdk/go/README.md

```diff
@@ -0,0 +1,101 @@
+# CDE - CloudEvents for Continuous Delivery
+
+Simple CLI and library to emit Cloud Events related to Continuous Delivery.
+
+The idea behind this framework and command-line interface is to empower you  emitting events related to a Continuous Delivery to enable interoperability between different tools, measure performance and gain visibility of your CD processes.
+
+
+## Usage
+
+Download the binary `cde` or build from source, by clonning this repository and run `make build`
+
+Set `CDE_SINK` environment variable to define where CloudEvents will be sent. You can do this by running:
+
+`export CDE_SINK=http://my-cloudevent-broker` 
+
+You can use [SockEye](https://github.com/n3wscott/sockeye) to visualize the cloud events.
+
+Integrate with your existing pipelines, repositories, and environments.
+
+## Supported Events
+
+Use `cde --help`
+
+The following events are supported:
+- [Environment Events](#environment-events)
+- [Service Events](#service-events)
+- [PipelineRun Events](#pipelinerun-events)
+- [TaskRun Events](#taskrun-events)
+- [Repository Events](#repository-events)
+            
+
+# CDE Events
+
+The following events are currently supported: 
+
+## Environment Events
+- Properties
+    - Id
+    - Name
+    - URL
+      
+Examples: 
+
+- **cd.environment.created.v1** `./cde env created --id staging --name "Staging Environment" --repo http://github.com/user/myrepo --data event=data`
```

Comment:
Not at all important for the PoC, but we should think about how to specify what version of the event we want to send, if we have multiple.

Maybe `cde env created.v1`

Reply:
@erkist this is a good question.. I would be more in favour of setting a global variable, for example CDE_VERSION=v1 or something like that in a global context to specify which version of the events will be emitted by the command-line tool. I don't think that we should cover cases where you want to send two events with different versions one after the other.. sounds like a bit too much. @erkist wdyt?